astropy/utils/tests/test_data_info.py

```diff
@@ -11,15 +11,15 @@
 from ..data_info import dtype_info_name
 from ...tests.helper import pytest
 
-STRING_TYPE_NAMES = {(False, 'S'): 'str',  # not PY3
+STRING_TYPE_NAMES = {(False, 'S'): 'str',  # PY2
                      (False, 'U'): 'unicode',
-                     (True, 'S'): 'bytes', # PY3
+                     (True, 'S'): 'bytes', # not PY2
```

Comment:
I'd prefer the PY3 here :smile: Any specific reason why you altered the comments?